model/src/main/kotlin/config/AdvisorConfiguration.kt

```diff
@@ -57,3 +58,15 @@ data class NexusIqConfiguration(
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     val password: String?
 ) : AdvisorConfiguration()
+
+/**
+ * The configuration for VulnerableCode as security vulnerability provider.
+ *
+ * TODO: Define options for authentication.
+ */
+data class VulnerableCodeConfiguration(
+    /**
+     * The base URL of the VulnerableCode REST API.
+     */
+    val vulnerableCodeServerUrl: String
```

Comment:
Please drop "vulnerableCode" and name this just "serverUrl" like in the `NexusIq` class. This whole class is about VulnerableCode, so we do not need that prefix.

Reply:
Yes, this is correct. We must, however, be careful here because of the way the configuration is loaded. We use the sealed classes approach to distinguish between different configuration classes. This only works if all classes are structural different. So if we introduced another configuration class with just a _serverUrl_ property, there would be a clash. This is not a problem yet, but just wanted to mention it.

Reply:
Thanks for pointing out that potential issue. However, I'd still prefer this to be named just `serverUrl` for now, and think about a solution once we really run into the problem.